docs/site/src/pages/reference/console/channels.mdx

```diff
@@ -0,0 +1,90 @@
+---
+layout: "@/layouts/MainLayout.astro"
+title: "Channels"
+heading: "Channels"
+description: "Store data in channels."
+---
+
+import { Divider } from "@synnaxlabs/pluto";
+import { Video } from "@/components/Media";
+import Table from "@/components/Table.astro";
+
+A [channel](/reference/concepts/channels) is a logical collection of time-ordered data
```

Comment:
what does logical in "logical collection" actually add to the explanation? i wouldn't have it there